src/RelativeStream.php

```diff
@@ -105,9 +103,9 @@ public function isSeekable()
     /**
      * {@inheritdoc}
      */
-    public function seek($offset, $whence = SEEK_SET)
+    public function seek($offset, $whence = \SEEK_SET)
     {
-        if ($whence == SEEK_SET) {
+        if ($whence === \SEEK_SET) {
```

Comment:
I would go with import the constant as we do in other places for internal functions:
```php
use const SEEK_SET;
```